src/main/java/Calculator.java

```diff
@@ -0,0 +1,5 @@
+public class Calculator {
+    private int add(int a, int b) {
+        return a + b;
+    }
+}
```

Comment:
```suggestion
}

```
it is a good practice to leave a blank line at the end of the class